src/js/protocols/stm32usbdfu.js

```diff
@@ -156,7 +156,9 @@ STM32DFU_protocol.prototype.claimInterface = function (interfaceNumber) {
     var self = this;
 
     chrome.usb.claimInterface(this.handle, interfaceNumber, function claimed() {
-        if(self.checkChromeError()) {
+        // Don't perform the error check on MacOS at this time as there seems to be a bug
+        // where it always reports an error even though it works correctly.
+        if (self.checkChromeError() && (GUI.operating_system !== "MacOS")) {
```

Comment:
Since the type of error that is thrown in MacOS is still logged to the console, it might make sense to add the type of error to the comment, so somebody else investigating a problem will know if this is the same or something different.

Reply:
I'm not sure "Error claiming interface" is too helpful, but ok.

Reply:
Not very informative - but if it helps somebody eliminate that an error they are getting is not the one classified as 'has no impact' then that's helpful.

Reply:
Updated the comment.